packages/util-ts/package.json

```diff
@@ -17,6 +17,7 @@
   "license": "MIT",
   "main": "./src/index.js",
   "peerDependencies": {
+    "type-fest": "^4",
```

Comment:
Is this really a peer dep? If we're only using it for its types, they may get stripped in the published lib. Doesn't hurt to include, I guess.

Reply:
I think so, because we're exporting a function with one of its types. Also, it's just auto-correcting and adding it to `dependencies` if I don't:
![image](https://github.com/user-attachments/assets/8aa4aea1-e2f1-43d1-a3ec-ed0780e3e489)